CHANGELOG/CHANGELOG-3.7.md

```diff
@@ -6,6 +6,14 @@ Previous change logs can be found at [CHANGELOG-3.6](https://github.com/etcd-io/
 
 ## v3.7.0 (TBD)
 
+### Breaking Changes
+
+- [Removed all deprecated experimental flags](https://github.com/etcd-io/etcd/pull/19959)
+
+### Package `pkg`
+
+- [Optimize find performance by splitting intervals with the same left endpoint by their right endpoints](https://github.com/etcd-io/etcd/pull/19768)
+
 ### Dependencies
 
 - Compile binaries using [go 1.24.2](https://github.com/etcd-io/etcd/pull/19723)
```

Comment:
Nit: Could we also update this to `1.24.3` to align with the branch current state?

Reply:
updated & thx